src/store-features/user.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const user = createSlice({
+    name: 'user',
+    initialState: {
+        email: '',
+        name: '',
+        profileImage: '',
+    },
+    reducers: {
+        setUser: (state, action) => {
+            const { email, name, profileImage } = action.payload;
+            state.email = email;
```

Comment:
email=email

Reply:
Resolved this by using object.assign
Object.assign(state, action.payload);